test/blackbox-tests/test-cases/bisect-ppx/github3473/run.t

```diff
@@ -0,0 +1,25 @@
+  $ cat >dune-project <<EOF
+  > (lang dune 2.5)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (library
+  >  (name hello)
+  >  (bisect_ppx))
+  > EOF
+
+  $ cat >dune-workspace <<EOF
+  > (lang dune 2.6)
+  > 
+  > (context default)
+  > (context (default (name coverage) (bisect_enabled true)))
+  > EOF
+
+  $ dune build @all 2>&1 | grep -v 'file "'
+  File "dune", line 3, characters 1-13:
+  3 |  (bisect_ppx))
+       ^^^^^^^^^^^^
+  Error: 'bisect_ppx' is available only when bisect_ppx is enabled in the
+  dune-project file. It cannot be enabled automatically because the currently
+  selected version of dune (2.5) does not support this plugin.
```

Comment:
It may be useful to tell the user the minimum version of dune lang required for the extension to be available ?

Reply:
That makes sense.